### ISSUE TITLE
Agent - low data help A. D. Doerge

### ISSUE THREAD

**Issue by ewommack** [ArctosDB/arctos#4925]
@acdoll @ccicero @cefilipek- Working on the low quality agents and came across this one that all our collections share:
https://arctos.database.museum/info/agentActivity.cfm?agent_id=10003273

Can any of us find any additional information on this person? 
I've found mention of them in publications, but just that they collected eggs, and still just by their initials and active in TX. Anything we can add to relationships or address or birth and death dates?

**Comment by ccicero** [comment]
I don't know anything about this collector. Sorry.

**Comment by ewommack** [comment]
Anything at all - associate of, employee of, died etc. If we can't get more out of him he'll go to verbatim.

**Comment by ccicero** [comment]
Why verbatim? We don't want to change the agent to something else, at least we have a last name. I'm not sure I understand the issue?

**Comment by ewommack** [comment]
This is part of the big agent clean up that Arctos is going to be doing: https://github.com/ArctosDB/arctos/issues/4903

Any type of information that we can add that can help move the Agent to a high quality agent will help. I've found mention of them in publications, but those are also just mention with the initials and last name.

**Comment by Jegelewicz** [comment]
This person appears to have been a [collector in Navasota, Texas](https://books.google.com/books?id=PeIyAQAAMAAJ&pg=RA1-PA44&lpg=RA1-PA44&dq=A.+D.+DOERGE+,+Navasota+,+Texas&source=bl&ots=7H26qWiMaU&sig=ACfU3U2Ftw6f7QmnB28CpNbP9CXCtBSaDQ&hl=en&sa=X&ved=2ahUKEwj70uHbpr35AhVbATQIHZgcCxkQ6AF6BAgUEAM#v=onepage&q=A.%20D.%20DOERGE%20%2C%20Navasota%20%2C%20Texas&f=false). When I find this stuff - I add the address "Navasota, Texas" and here you know at least he was there in "Winter of 1906" when this publication was printed.

I added all of this to the agent - it should be good to go now.

He is listed as a member of the American Society of Curio Collectors in the same issue.

**Comment by cjconroy** [comment]
They have a note in the Oologist in 1905. Says they are from Navasota, TX. 
https://www.google.com/books/edition/The_O%C3%B6logist/ZNEyAQAAMAAJ?hl=en&gbpv=1&dq=A.+D.+Doerge+eggs&pg=RA4-PA26&printsec=frontcover

https://www.findagrave.com/memorial/118399626/albert-dalton-doerge

**Comment by ewommack** [comment]
Thanks @Jegelewicz and @cjconroy!

Great to know we can use simple localities for addresses when we have to. Additional information for how to solve and get these agents into better condition.

**Comment by dustymc** [comment]
> can use simple localities for addresses when we have to

Yes, maybe an item for the newsletter - address "Texas" or alive=1905 is well above the "more than strings can do" bar, and both of those together starts to look like real data!

I updated preferred name from @cjconroy 's link - thanks!

I changed '[dead](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctAgent_Status#dead)' to '[died](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctAgent_Status#died)' - I've been using the latter in reports, I think born/died can generally DO STUFF better than dead/alive dates, maybe that needs clarified as well.
